services/src/main/java/org/keycloak/protocol/oidc/OIDCAdvancedConfigWrapper.java

```diff
@@ -74,7 +74,23 @@ public void setRequestObjectSignatureAlg(Algorithm alg) {
         String algStr = alg==null ? null : alg.toString();
         setAttribute(OIDCConfigAttributes.REQUEST_OBJECT_SIGNATURE_ALG, algStr);
     }
-    
+
+    public void setRequestObjectEncryptionAlg(String algorithm) {
+        setAttribute(OIDCConfigAttributes.REQUEST_OBJECT_ENCRYPTION_ALG, algorithm);
+    }
+
+    public String getRequestObjectEncryptionAlg() {
+        return getAttribute(OIDCConfigAttributes.REQUEST_OBJECT_ENCRYPTION_ALG);
+    }
+
+    public String getRequestObjectEncryptionEncAlg() {
+        return getAttribute(OIDCConfigAttributes.REQUEST_OBJECT_ENCRYPTION_ENC_ALG);
+    }
+
+    public void setRequestObjectEncryptionEncAlg(String algorithm) {
```

Comment:
If https://github.com/keycloak/keycloak/pull/8261/files#r672633565 is agreed, it might be better to be `setRequestObjectEncryptionEnc` .